scss/_base_typography.scss

```diff
@@ -89,30 +89,36 @@
   }
 
   //@section Adjusted spacing for headings (or a p) following a paragraph
-  p:not([class*='p-heading--']):not([class*='p-muted-heading']) {
+  p:not([class*='p-heading-']):not([class*='p-muted-heading']) {
```

Comment:
Not going for the double dash in this selector?

Reply:
Technically either would work, but good catch, I searched/replaced only the one with numbers, but forgot about those. Will fix.